tests/relational/test_train_transforms.py

```diff
@@ -0,0 +1,85 @@
+import tempfile
+from unittest.mock import ANY, patch
+
+import pytest
+
+from gretel_trainer.relational.core import MultiTableException
+from gretel_trainer.relational.multi_table import MultiTable
+
+
+# The assertions in this file are concerned with setting up the transforms train
+# workflow state properly, and stop short of kicking off the task.
+@pytest.fixture(autouse=True)
+def run_task():
+    with patch("gretel_trainer.relational.multi_table.run_task"):
+        yield
+
+
+@pytest.fixture(autouse=True)
+def backup():
+    with patch.object(MultiTable, "_backup", return_value=None):
+        yield
+
+
+@pytest.fixture()
+def tmpdir(project):
+    with tempfile.TemporaryDirectory() as tmpdir:
+        project.name = tmpdir
+        yield tmpdir
+
+
+def test_train_transforms_defaults_to_transforming_all_tables(ecom, tmpdir):
+    mt = MultiTable(ecom, project_display_name=tmpdir)
+    mt.train_transforms("transform/default")
+    transforms_train = mt._transforms_train
+
+    assert set(transforms_train.models.keys()) == set(ecom.list_all_tables())
+
+
+def test_train_transforms_only_includes_specified_tables(ecom, tmpdir, project):
+    mt = MultiTable(ecom, project_display_name=tmpdir)
+    mt.train_transforms("transform/default", only=["users"])
```

Comment:
Q: Is it possible to run `train_transforms` multiple times? For example I have 2 tables and want to use different config for each one:

```python
mt = MultiTable(ecom, project_display_name=tmpdir)
mt.train_transforms("transform/default", only=["users"])
mt.train_transforms("transform/redact-pii-nlp-off", only=["purchases"])
```

Would something like this work?

Reply:
Yes! If you called `mt.run_transforms()` after that snippet you wrote, you'll kick off jobs for both users and purchases. Also worth noting that if you called `train_transforms` twice and some table was included in both calls (e.g. first `only=["users"]` and then `only=["users", "purchases"]`), we'll essentially drop the first model on the floor in terms of MultiTable's internal state. (It will remain in the project, though.)

Reply:
Nice, thanks!

Reply:
This felt worth capturing in a test